src/content/pages/accessibility.mdx

```diff
@@ -120,14 +120,14 @@ Registration will be on the **1st floor**
 ### Low stimulation room (South room 243)
 
 - If you feel overwhelmed and need a quiet space with no distraction feel free to visit **South Room 243** on the **2nd floor**. To get there look for childcare and keep on going through the hallway. It will be one of the doors to your left.
```

Comment:
If you feel overwhelmed and need a quiet space with no distractions, feel free to visit **South Room 243** on the **2nd floor**. To get there, look for the childcare room and continue through the hallway. It will be one of the doors to your left.